src/components/Settings/Settings.react.js

```diff
@@ -466,18 +468,20 @@ class Settings extends Component {
         /* eslint-enable */
         this.setState({
           dataFetched: true,
-          PrefLanguage: response.settings.prefLanguage,
-          avatarType: response.settings.avatarType,
-          UserName: response.settings.userName,
-          TimeZone: response.settings.timeZone,
-          countryCode: response.settings.countryCode,
-          CountryDialCode: response.settings.countryDialCode,
-          PhoneNo: response.settings.phoneNo,
-          EnterAsSend: response.settings.enterAsSend,
-          MicInput: response.settings.micInput,
-          theme: response.settings.theme,
-          SpeechOutput: response.settings.speechOutput,
-          SpeechOutputAlways: response.settings.speechOutputAlways,
+          PrefLanguage: response.settings && response.settings.prefLanguage,
```

Comment:
Checking if ```response.settings``` is valid for every var seems to be redundant. Could we improve here?